src/components/sbb-menu/sbb-menu.scss

```diff
@@ -0,0 +1,163 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-menu-position-x: 0;
+  --sbb-menu-position-y: 0;
+  --sbb-menu-min-height: #{px-to-rem-build(120)};
+  --sbb-menu-max-width: #{px-to-rem-build(320)};
+  --sbb-menu-min-width: #{px-to-rem-build(240)};
+}
+
+// stylelint-disable plugin/stylelint-bem-namics
+::slotted(:not(sbb-menu-action)) {
+  display: block;
+  padding-inline: var(--sbb-spacing-fixed-5x);
+}
+// stylelint-enable plugin/stylelint-bem-namics
+
+::slotted(sbb-divider) {
+  --divider-color: var(--sbb-color-iron-default);
+
+  margin: var(--sbb-spacing-fixed-2x) 0;
+}
+
+.sbb-menu {
+  @include scrollbar('dark');
+
+  --sbb-menu-transform: translateY(100%);
+
+  max-height: calc(100vh - var(--sbb-spacing-fixed-8x));
+  min-width: 100%;
+  text-align: start;
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  top: unset;
+  right: unset;
+  color: var(--sbb-color-white-default);
+  border: none;
+  border-radius: var(--sbb-border-radius-4x) var(--sbb-border-radius-4x) 0 0;
+  background-color: var(--sbb-color-black-default);
+  padding-inline: 0;
+  padding-block: var(--sbb-spacing-fixed-1x);
+
+  &[open] {
+    animation: {
+      name: show;
+      duration: var(--sbb-animation-duration-6x);
+      timing-function: ease;
+    }
+
+    &.sbb-menu--no-animation {
+      animation-duration: 0s;
+    }
+  }
+
+  &::backdrop {
+    background: rgb(0 0 0 / 20%);
+
+    animation: {
+      name: show-backdrop;
+      duration: 0.24s;
+      timing-function: ease;
+    }
+  }
+
+  &.sbb-menu--no-animation {
+    &::backdrop {
+      animation-duration: 0s;
+    }
+  }
+
+  &::after {
+    content: '';
+    display: block;
+    height: var(--sbb-spacing-fixed-8x);
+  }
+
+  &.sbb-menu--dismissing {
+    animation-name: hide;
+
+    &::backdrop {
+      animation-name: hide-backdrop;
+    }
+  }
+}
+
+@include ifForcedColors {
+  dialog {
+    outline: var(--sbb-border-width-1x) solid CanvasText;
+  }
+}
+
+@include mq($from: medium) {
+  .sbb-menu {
+    --sbb-menu-transform: translateY(var(--sbb-spacing-fixed-2x));
+
+    position: absolute;
+    top: var(--sbb-menu-position-y);
+    left: var(--sbb-menu-position-x);
+    right: unset;
+    bottom: unset;
+    max-height: var(--sbb-menu-max-height);
+    min-height: var(--sbb-menu-min-height);
+    max-width: var(--sbb-menu-max-width);
+    min-width: var(--sbb-menu-min-width);
+    border-radius: var(--sbb-border-radius-4x);
+
+    &::backdrop {
+      background: transparent;
+      animation: none;
+    }
+
+    &::after {
+      display: none;
+    }
+  }
+}
+
+@keyframes show {
```

Comment:
It is necessary to use animations with keyframes instead of transitions in order not to alter the default `display: block` of the modal otherwise it causes several problems, especially for accessibility.